.github/workflows/remove-labels.yml

```diff
@@ -0,0 +1,19 @@
+name: Autoremove Labels
+
+on:
+  issues:
+    types: [closed]
+  pull_request_target:
+    types: [closed]
+
+jobs:
+
+  RemoveWaitingLabelFromClosedIssueOrPR:
+    if: github.event.action == 'closed'
```

Comment:
Since you've set this to only run when `issues` or `pull_request_target` are `closed`, you can omit this line (but it won't harm anything 🙂).